types/src/v0/traits.rs

```diff
@@ -461,6 +461,7 @@ pub trait SequencerPersistence: Sized + Send + Sync + 'static {
                 highest_voted_view,
                 saved_proposals,
                 high_qc,
+                None,
```

Comment:
I think we want a `load_upgrade_certificate` method in `SequencerPersistence` to be able to call in this function, so that we can pass the stored certificate here